src/metabase/api/public.clj

```diff
@@ -129,5 +133,19 @@
                            :dashboardcard_id [:in dashcard-ids])))))
   (run-query-for-card-with-id card-id parameters))
 
+(api/defendpoint GET "/oembed"
+  "oEmbed endpoint"
```

Comment:
Can you make this docstring a little bit more useful? Why do we need this endpoint / what's it used for / etc.